src/components/FilterDrinks.spec.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import FilterDrinks from './FilterDrinks';
+
+describe('FilterDrinks', () => {
+  it('render three radio buttons with three labels', () => {
+    render(<FilterDrinks currentFilter={'all'} />);
+
+    const radioButtonAll = screen.getByLabelText('All');
+    const radioButtonAlcoholic = screen.getByLabelText('Alcoholic');
+    const radioButtonNonAlcoholic = screen.getByLabelText('Non-Alcoholic');
+
+    expect(radioButtonAll).toBeInTheDocument();
+    expect(radioButtonAlcoholic).toBeInTheDocument();
+    expect(radioButtonNonAlcoholic).toBeInTheDocument();
+  });
+
+  it('when clicking on a radio button the onChange function is called', () => {
+    const radioCallback = jest.fn();
+    render(
+      <FilterDrinks handleChangeFilter={radioCallback} currentFilter={'all'} />
+    );
+
+    const radioButtonAlcoholic = screen.getByLabelText('Alcoholic');
+
+    userEvent.click(radioButtonAlcoholic);
+
+    expect(radioCallback).toBeCalled();
+  });
+});
```

Comment:
I like your tests! 👍 You should also test if the other buttons work, because your second test only tests one out of three radio buttons. :-)